{
  service: 'user-service',
  ok: 0,
  code: 8000,
  codeName: 'AtlasError',
  connectionGeneration: 0,
  level: 'error',
  message: 'faield to connect db bad auth : Authentication failed.',
  stack: 'MongoServerError: bad auth : Authentication failed.\n' +
    '    at Connection.onMessage (G:\\Level 2 PHero\\university-management-auth-backend\\node_modules\\mongodb\\src\\cmap\\connection.ts:413:18)\n' +
    '    at MessageStream.<anonymous> (G:\\Level 2 PHero\\university-management-auth-backend\\node_modules\\mongodb\\src\\cmap\\connection.ts:243:56)\n' +
    '    at MessageStream.emit (node:events:513:28)\n' +
    '    at processIncomingData (G:\\Level 2 PHero\\university-management-auth-backend\\node_modules\\mongodb\\src\\cmap\\message_stream.ts:187:12)\n' +
    '    at MessageStream._write (G:\\Level 2 PHero\\university-management-auth-backend\\node_modules\\mongodb\\src\\cmap\\message_stream.ts:68:5)\n' +
    '    at writeOrBuffer (node:internal/streams/writable:391:12)\n' +
    '    at _write (node:internal/streams/writable:332:10)\n' +
    '    at MessageStream.Writable.write (node:internal/streams/writable:336:10)\n' +
    '    at TLSSocket.ondata (node:internal/streams/readable:754:22)\n' +
    '    at TLSSocket.emit (node:events:513:28)',
  label: 'um',
  timestamp: '2023-06-06T05:49:04.404Z',
  [Symbol(errorLabels)]: Set(2) { 'HandshakeError', 'ResetPool' }
}
